test/cmdlineTests/truthy_bool_comparison_peephole/args

```diff
@@ -0,0 +1 @@
+--asm
```

Comment:
Gotta put newlines at the end of files. Before pushing, run `scripts/check_style.sh` to see if you need to make further changes.

Reply:
Oh yea, did try to run it but was getting this, which I still need to look into:
``` 
$ scripts/check_style.sh 
grep: repetition-operator operand invalid

$ sysctl kern.version
kern.version: Darwin Kernel Version 21.2.0: Sun Nov 28 20:28:41 PST 2021; root:xnu-8019.61.5~1/RELEASE_ARM64_T6000
$ grep -V
grep (BSD grep, GNU compatible) 2.6.0-FreeBSD
```

Reply:
Hmm... maybe the script is broken on macOS. I'd expect someone to report it if that was the case but on the other hand we rarely run it locally. It's executed by CI anyway and once you get the hang of the style, it's rare to trigger it in practice.

You could report it as a separate issue and we'll fix it.

Reply:
https://github.com/ethereum/solidity/issues/13492

Reply:
Looks like this was broken for ages; left a comment on the issue.